assets/samples/dynakube/v1beta3/logMonitoring.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: dynatrace.com/v1beta3
+kind: DynaKube
+metadata:
+  name: dynakube
+  namespace: dynatrace
+spec:
+  apiUrl: https://ENVIRONMENTID.live.dynatrace.com/api
+
+  # Link to api reference for further information: https://docs.dynatrace.com/docs/ingest-from/setup-on-k8s/reference/dynakube-parameters#tabgroup--apiversions--v1beta2
+
+# This feature is still experimental, for more context please refer to: https://docs.dynatrace.com/docs/ingest-from/setup-on-k8s/reference/dynakube-parameters
+  logMonitoring: {}
+
+  templates: {}
+    # Required in combination with logMonitoring enabled.
+    # As there is no image available in public registry yet, this field is required if you want to run logMonitoring.
+    #
+    # logMonitoring:
+    #   imageRef:
+    #     repository: <image-repo>
+    #     tag: <image-tag>
```

Comment:
I see a ⛔  sign here, why 😅 ?

Reply:
![image](https://github.com/user-attachments/assets/4b2b979b-82b3-4440-a526-1e70c05d60f9)

Reply:
maybe because no blank line in the end? lemme try

Reply:
didnt fix it https://github.com/Dynatrace/dynatrace-operator/pull/4160/commits/a15b94b967aaef608cfd227bd5065f1fdee71b94

dunno tbh

Reply:
that is the sign for missing line at the end of file though 🙂

Reply:
isn't that just an EOF char?

Reply:
https://github.com/Dynatrace/dynatrace-operator/pull/4160/commits/ecec76730f4a9997d83f806186b03a48d0451993 this fixed it. I removed the trailing tab as well.